tools/solidity-coverage-example/README.md

```diff
@@ -0,0 +1,34 @@
+# Sample Hardhat Project
+
+This project demonstrates a basic Hardhat use case with Hedera. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
+
+Try running some of the following tasks:
+
+```shell
+npx hardhat help
+npx hardhat test
+REPORT_GAS=true npx hardhat test
+npx hardhat node
+npx hardhat run scripts/deploy.ts
+```
+
+## Hedera and Hardhat integration
+
+You can find a npm script that launches a http server to serve the code coverage report.
+But first the reports have to be generated:
+```shell
+// to run the tests and generate the reports
+npm run test
+// serve the reports on http://127.0.0.1:8080 by default
+npm run coverage:serve
```

Comment:
Can you add a print out or image of what a a successful run looks like maybe for the `Lock.sol`

Reply:
Done